sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -30,22 +32,19 @@ public async Task RunSampleAsync()
             ModuleIdentity moduleIdentity = await CreateModuleIdentityAsync(deviceIdentity, moduleId);
 
             // List all modules within the device.
-            await ListAllModulesAsync();
+            await ListAllModulesAsync(deviceIdentity);
 
             // Get the Module Identity.
-            await GetModuleIdentityAsync();
+            moduleIdentity = await GetModuleIdentityAsync(deviceIdentity, moduleIdentity.ModuleId);
```

Comment:
why wouldn't we use `moduleId` instead of `moduleIdentity.ModuleId`?
`_moduleIdentity_ = await GetModuleIdentityAsync(deviceIdentity, _moduleIdentity.ModuleId_);` seems like a cylic-reference.

Reply:
If we require only the device Id and module Id, why not pass in `deviceId` and `moduleId`?

Reply:
The main idea was to pass in the device identity object and show what it contains and what can be done with the object. 
I don't feel that strongly about it and if you think passing down the minimum required parameters looks better and is easier to follow, I will change them.